target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch

```diff
@@ -581,12 +580,11 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +	xt_flowoffload_check_device(table, devs[0]);
 +	xt_flowoffload_check_device(table, devs[1]);
 +
-+	dst_release(route.tuple[!dir].dst);
-+
 +	return XT_CONTINUE;
 +
 +err_flow_add:
 +	flow_offload_free(flow);
++	dst_release(route.tuple[dir].dst);
 +err_flow_alloc:
```

Comment:
Please move `dst_release(route.tuple[dir].dst);` down one line. We expect the same changes as the upstream commit. And a run test would be better. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.80&id=9c5662e95a8dcc232c3ef4deb21033badcd260f6
```
err_flow_alloc:
	dst_release(route.tuple[dir].dst);
	dst_release(route.tuple[!dir].dst);
```

Reply:
I moved that line but I get an error when I try to refresh:
```
Applying patch generic-hack/650-netfilter-add-xt_FLOWOFFLOAD-target.patch
patching file net/netfilter/Kconfig
patching file net/netfilter/Makefile
patching file net/netfilter/xt_FLOWOFFLOAD.c
patch: **** malformed patch at line 737: --- a/net/netfilter/nf_flow_table_core.c

Patch generic-hack/650-netfilter-add-xt_FLOWOFFLOAD-target.patch does not apply (enforce with -f)
make[3]: *** [Makefile:16: /scratch/stable-openwrt/build_dir/target-arm_cortex-a7_musl_eabi/linux-airoha_generic/linux-6.1.80/.quilt_checked] Error 1
make[3]: Leaving directory '/scratch/stable-openwrt/target/linux/airoha'
make[2]: *** [Makefile:11: refresh] Error 2
make[2]: Leaving directory '/scratch/stable-openwrt/target/linux'
time: target/linux/refresh#11.32#6.85#14.30
    ERROR: target/linux failed to build.
make[1]: *** [target/Makefile:30: target/linux/refresh] Error 1
make[1]: Leaving directory '/scratch/stable-openwrt'
make: *** [/scratch/stable-openwrt/include/toplevel.mk:233: target/linux/refresh] Error 2
```

Reply:
I guess the previous version should works. https://github.com/openwrt/openwrt/commit/1ce0d59a299e475887c40445c2bf34a4f4e3ec58